src/promptflow/promptflow/_sdk/_service/app.py

```diff
@@ -137,9 +137,11 @@ def log_before_request_info():
             else:
                 request_body = request.get_data()
 
+            app.logger.info(
+                "Request coming in: %s, last request time: %s", request.url, app.config["last_request_time"]
```

Comment:
Use a variable, not read the app.config. Will app config have concurrent issue?

Reply:
app.config is a global variable when multiple http request call, I think the last request time should be the last http request which I think is what's expected here.

remove info message here 
![image](https://github.com/microsoft/promptflow/assets/26239730/76ffb95c-f2bc-4553-aded-e895e3967d3c)